core/server/master/src/main/java/alluxio/master/file/InodeTtlChecker.java

```diff
@@ -71,16 +71,17 @@ public void heartbeat() {
         if (path != null) {
           try {
             TtlAction ttlAction = inode.getTtlAction();
-            LOG.debug("Path {} TTL has expired, performing action {}", path.getPath(), ttlAction);
+            LOG.info("Path {} TTL has expired, performing action {}", path.getPath(), ttlAction);
             switch (ttlAction) {
               case FREE:
                 // public free method will lock the path, and check WRITE permission required at
                 // parent of file
                 if (inode.isDirectory()) {
-                  mFileSystemMaster
-                      .free(path, FreeOptions.defaults().setForced(true).setRecursive(true));
+                  mFileSystemMaster.delete(path,
+                      DeleteOptions.defaults().setRecursive(true).setAlluxioOnly(true));
```

Comment:
Temporarily solution. I filed a jira to describe it, for more information please see [ALLUXIO-3182](https://alluxio.atlassian.net/browse/ALLUXIO-3182)